### PR TITLE
Oo/oslynn/12999/onboarding epic sender and receiver

### PR DESCRIPTION
**Pull request by oslynn**
This PR onboards Epic as sender and receiver.

Test Steps:
1. *Include steps to test these changes*

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

### Testing
- [x] Tested locally?
- [x] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue
- https://app.zenhub.com/workspaces/engagement-6166edbd409257001e09f1b2/issues/gh/cdcgov/prime-reportstream/12999

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

File: prime-router/settings/staging/0169-epic-staging-sender-receiver.yml
Comment (basiliskus) [inline prime-router/settings/staging/0169-epic-staging-sender-receiver.yml:39]: The schema here will be `metadata/hl7_mapping/receivers/Flexion/TIResults` once PR #13308 gets merged. BTW, that PR is ready for review so you can take a look and approve if everything looks good to you

File: prime-router/settings/staging/0169-epic-staging-sender-receiver.yml
Comment (basiliskus) [inline prime-router/settings/staging/0169-epic-staging-sender-receiver.yml:41]: `useBatchHeaders` should be `false`

File: prime-router/settings/staging/0169-epic-staging-sender-receiver.yml
Comment (basiliskus) [inline prime-router/settings/staging/0169-epic-staging-sender-receiver.yml:84]: This routing filter will change once we get clarity on what the id will be for the receiver. The filter will be something like this: `Bundle.entry.resource.ofType(MessageHeader).destination.receiver.resolve().identifier.where(value = 'receiver-id').exists()` where we'll replace `receiver-id`

File: prime-router/settings/staging/0169-epic-staging-sender-receiver.yml
Comment (basiliskus) [inline prime-router/settings/staging/0169-epic-staging-sender-receiver.yml:85]: The system url is `http://localcodes.org/ETOR` instead of `http://etoruri`

**Review by basiliskus — CHANGES_REQUESTED** [review]
Thanks Ott, left a few comments with corrections

File: prime-router/settings/staging/0169-epic-staging-sender-receiver.yml
Comment (basiliskus) [reply prime-router/settings/staging/0169-epic-staging-sender-receiver.yml:39]: FYI, I removed `TIResults` schema for now, so you can leave the schema as is. No need to update it

**Comment by GilmoreA6** [comment]
Closing PR as it is no longer needed

**Pull request closed without merging**
